### PR TITLE
Add explicit Python requirements to tools that appear to require it

### PR DESCRIPTION
**Pull request by natefoo**

### CODE REVIEW

File: tools/ncbi_blast_plus/ncbi_makeblastdb.xml [hunk 1]
```diff
@@ -4,7 +4,9 @@
         <token name="@BINARY@">makeblastdb</token>
         <import>ncbi_macros.xml</import>
     </macros>
-    <expand macro="preamble" />
+    <expand macro="preamble">
+        <requirement type="package" version="3.9">python</requirement>
+    </expand>
```
Comment (peterjc) [inline tools/ncbi_blast_plus/ncbi_makeblastdb.xml:9]: The macros can do that can they?

**Review by peterjc — APPROVED** [review]
That extended macro stuff is rather neat.

I really ought to get CI working here again though...

**Comment by natefoo** [comment]
Thanks Peter!

**Comment by bgruening** [comment]
Mh, it seems this was never deployed. I don't see 0.1.2 of [blastxml_to_top_descr.xml](https://github.com/peterjc/galaxy_blast/pull/142/files#diff-98e5d8a6bca3f850afc2b9946a31e941e7139ea65cb9ac1ebd6d7e29bf3a2ea7) in the TS.

**Comment by peterjc** [comment]
Looks like it was done between TravisCI and GitHub Actions automation...

**Comment by peterjc** [comment]
I have updated the tool sheds manually for https://github.com/natefoo/galaxy_blast/tree/master/tools/blastxml_to_top_descr

**Pull request merged**